src/css/custom.css

```diff
@@ -96,8 +96,8 @@ table {
 
 .buy-now-nav-item:hover {
   background: var(--theme-color-white) !important;
-  color: var(--theme-color-dark-navy-blue) !important;
-  border: 2px solid var(--theme-color-dark-navy-blue) !important;
+  color: var(--theme-color-stormy-blue) !important;
+  border: 2px solid var(--theme-color-stormy-blue) !important;
```

Comment:
`!important` needed only for `color`

Reply:
Okay, now I see if we want to delete it then we need to add another !important somewhere else :dizzy: